test/battle/ability/parental_bond.c

```diff
@@ -24,23 +24,166 @@ SINGLE_BATTLE_TEST("Parental Bond converts Tackle into a two-strike move")
     }
 }
 
+SINGLE_BATTLE_TEST("Parental Bond does not convert Triple Kick into a two-strike move")
+{
+    GIVEN {
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].split != SPLIT_STATUS);
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].strikeCount == 3);
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].effect == EFFECT_TRIPLE_KICK);
+        PLAYER(SPECIES_KANGASKHAN) { Item(ITEM_KANGASKHANITE); }
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_TRIPLE_KICK, megaEvolve: TRUE); MOVE(opponent, MOVE_CELEBRATE); }
+    } SCENE {
+        MESSAGE("Kangaskhan's Kangaskhanite is reacting to 1's Mega Ring!");
+        ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_MEGA_EVOLUTION, player);
+        MESSAGE("Kangaskhan has Mega Evolved into Mega Kangaskhan!");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_TRIPLE_KICK, player);
+        HP_BAR(opponent);
+        HP_BAR(opponent);
+        HP_BAR(opponent);
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CELEBRATE, opponent);
+    } THEN {
+        EXPECT_EQ(player->species, SPECIES_KANGASKHAN_MEGA);
+    }
+}
+
+SINGLE_BATTLE_TEST("Parental Bond converts multi-target moves into a two-strike move in Single Battles")
+{
+    GIVEN {
+        ASSUME(gBattleMoves[MOVE_EARTHQUAKE].split != SPLIT_STATUS);
+        ASSUME(gBattleMoves[MOVE_EARTHQUAKE].strikeCount < 2);
+        ASSUME(gBattleMoves[MOVE_EARTHQUAKE].effect == EFFECT_EARTHQUAKE);
```

Comment:
Again not sure if assuming Earthquake effect matters here but I think the assume for `MOVE_TARGET_FOES_AND_ALLY` should be added.